client/beadledom-client-test/pom.xml

```diff
@@ -38,6 +38,11 @@
     </dependency>
 
```

Comment:
Are you intending this to go in both 3.x and 4.x of Beadledom?

Reply:
No this is only intended for 4.x release